docs/UserGuide.md

```diff
@@ -150,18 +159,19 @@ Format: `list`
 
 Edits an existing person in the address book.
```

Comment:
I think we can change "person" to "contact" to be consistent with the terminology we use.

Reply:
If this is okay, perhaps we could also look into other parts of the UG that say person instead of contact and replace those instances accordingly?